services/movie_session.py

```diff
@@ -1,11 +1,10 @@
 from django.db.models import QuerySet
+from db.models import MovieSession, Ticket
 
-from db.models import MovieSession
 
-
-def create_movie_session(
-    movie_show_time: str, movie_id: int, cinema_hall_id: int
-) -> MovieSession:
+def create_movie_session(movie_show_time: str,
```

Comment:
Consider using `datetime` objects for `movie_show_time` instead of strings to ensure type safety and avoid potential parsing errors.